.murdock

```diff
@@ -365,14 +365,22 @@ compile() {
         fi
     fi
 
+    # log some build stats
     if [ -d ${BINDIR} ]
     then
-        echo "-- build directory size: $(du -sh ${BINDIR} | cut -f1)"
-
+        echo "{\"build\": { \"bin/\": $(du -sh ${BINDIR} | cut -f1)}}"
         # cleanup
         rm -rf ${BINDIR}
     fi
 
+    BUILDDIR=build/
+    if [ -d ${BUILDDIR} ]
+    then
+        echo "{\"build\": { \"build/\": $(du -sh ${BUILDDIR} | cut -f1)}}"
+        # cleanup
+        #rm -rf ${BUILDDIR}
```

Comment:
Are these needed?

Reply:
no. I just realized that BINDIR is "build/" (same as BUILDDIR). I've removed the whole BUILDDIR block.